src/main/frontend/components/container.css

```diff
@@ -621,15 +619,95 @@ html[data-theme='dark'] {
     margin-left: 12px;
   }
 
+  .sidebar-drop-indicator {
+    @apply relative;
+    height: 8px;
+
+    &::after {
+      @apply absolute block w-full rounded;
+      top: 2px;
+      height: 4px;
+      content: " ";
+      transition: background-color 200ms;
+    }
+
+    &.drag-over {
+      &::after {
+        background-color: var(--ls-active-primary-color);
+      }
+    }
+  }
+
   .sidebar-item {
-    @apply p-4;
+    @apply relative;
+    flex: 1 1;
+    min-height: 100px;
+    border: 1px solid transparent;
```

Comment:
The border is visible in the light theme, is that expected?
![CleanShot 2023-08-03 at 10 45 50](https://github.com/logseq/logseq/assets/479169/183cc4c8-81c7-42bb-9062-ae26ea938291)

In the dark theme, it looks like this:
![CleanShot 2023-08-03 at 10 46 31](https://github.com/logseq/logseq/assets/479169/93fadbc1-2287-4ad7-8939-5e0a980e041e)